aptos-move/block-executor/src/executor.rs

```diff
@@ -1387,12 +1387,19 @@ fn resource_group_error(err_msg: String) -> PanicOr<IntentionalFallbackToSequent
 }
 
 fn gen_id_start_value(sequential: bool) -> u32 {
-    // IDs are ephemeral. Pick a random prefix, and different each time,
-    // in case exchange is mistakenly not performed - to more easily catch it.
-    // And in a bad case where it happens in prod, to and make sure incorrect
-    // block doesn't get committed, but chain halts.
-    // (take a different range from parallel execution, to even more easily differentiate)
-
+    // All identifiers are ephemeral. We pick a fixed large value to start with
```

Comment:
This should never happen - and we need to make sure that is the case. 

But if it does - it is better to halt the chain, than store incorrect values. (Imagine random large identifier being stored as account balance). It is also easier to catch halting issues in testing, than incorrect value. 

Can you revert this change from this PR?

Reply:
Also making starting point fixed, doesn't make them deterministic,  as execution order in BlockSTM dictates which value gets which identifier. So this just makes it more likely to be identical in small tests, which is opposite of what we want